bin/project/token/edit/index.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const Cli = require('lib/cli');
+const inquirer = require('inquirer');
+const yaml = require('js-yaml');
+
+
+module.exports = resource => {
+    const options = {
+        [resource.name]: {
+            description: `${resource.title} ID`,
+            type: 'string',
+            required: true,
+        },
+    };
+
+    return Cli.createCommand('edit', {
+        description: `Edit ${resource.title}`,
+        resource: resource,
+        dirname: __dirname,
+        options: Object.assign({}, options, resource.options),
+        handler: async args => {
+            const url = `${resource.url(args)}/${args[resource.name]}`;
+            const result = await args.helpers.api.get(url);
+
+            const questions = {
+                type: 'editor',
+                name: 'input',
+                message: 'Specify the name and access policy of the token.',
+                default: yaml.safeDump({
+                    name: result.name,
+                    access: result.access.map(access => ({
+                        method: access.method,
+                        path: access.path,
+                    })),
+                }),
+                validate: function (text) {
+                    try {
+                        yaml.safeLoad(text);
+                    } catch (e) {
+                        return 'You can not change the data format (YAML). The data entered is not a valid YAML document.';
```

Comment:
Does the error contains some helpful information?

Reply:
Yes.